Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM rocker/r-ver:4.3.1
+
+ENV RENV_CONFIG_REPOS_OVERRIDE "https://cloud.r-project.org/"
+ENV RENV_PATHS_LIBRARY renv/library
+
+# Install system dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    libcurl4-openssl-dev libssl-dev libxml2-dev libgit2-dev git \
+    libudunits2-dev python3-dev python3-pip
+
+# Install pipenv for Python dependencies
+RUN pip install pipenv
+
+# Install renv for R dependencies
+RUN Rscript -e "install.packages('renv')"
+
+# Copy pipenv files into the image. The reason this is a separate step from
+# the later step that adds files from the working directory is because we want
+# to avoid having to reinstall dependencies every time a file in the directory
+# changes, as Docker will bust the cache of every layer following a layer that
+# needs to change
+COPY Pipfile .
+COPY Pipfile.lock .
+
+# Install Python dependencies
+RUN pipenv install --system --deploy
```

Comment:
**issue (maybe blocking):** We might be able to dump a huge proportion of the pip requirements, or even drop them entirely, depending on how things will run on AWS.

Basically, the only things using Python in this repo are sales val (which will shortly be removed) and DVC. If AWS is going to use the container image directly, then we should keep pip/DVC. If AWS is going to use a separate runner and then use the Docker container only to run the R pipeline, then we can remove the pip reqs here and move them to the runner.

Either way, we should only need to install the boto + DVC stuff from here on out.

Reply:
Awesome, I switched to just installing `dvc[s3]` in `Pipfile` in 62c144a00373ad12ee26118b4331313b91f40988 and that seems to keep DVC and the S3 stuff 👍🏻